### PR TITLE
More example workflows

### PR DESCRIPTION
**Pull request by aniehuser**

### CODE REVIEW

File: demos/example-workflows/dataframe_work.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# The purpose of this script is to exchange links stored in a dataframe of players and
```
Comment (clundin55) [inline demos/example-workflows/dataframe_work.py:1]: This is already in the NBA workflow

Comment (clundin55) [reply demos/example-workflows/dataframe_work.py:1]: sorry thought this was new, just saw you only moved parent folders

**Review by clundin55 — APPROVED** [review]
I'm confused whether these are new or this just a new directory

**Comment by aniehuser** [comment]
@clundin55 some are new, some are not. I thought it would be nice to consolidate workflows in a single directory

**Pull request merged**